src/OrbitQt/main.cpp

```diff
@@ -257,6 +258,30 @@ static bool DevModeEnabledViaEnvironmentVariable() {
   return env.contains("ORBIT_DEV_MODE") || env.contains("ORBIT_DEVELOPER_MODE");
 }
 
+static void LogAndMaybeWarnAboutClockResolution() {
+  uint64_t estimated_clock_resolution = orbit_base::EstimateClockResolution();
+  LOG("%s",
+      absl::StrFormat("Clock resolution on client: %" PRIu64 " (ns)", estimated_clock_resolution));
+
+  // Since a low clock resolution on the client only affects our own introspection and logging
+  // timings, we only show a warning dialogue when running in devmode.
+  constexpr uint64_t kWarnThresholdClockResolutionNs = 10 * 1000;  // 10 us
+  if (absl::GetFlag(FLAGS_devmode) &&
+      estimated_clock_resolution > kWarnThresholdClockResolutionNs) {
+    DisplayErrorToUser(
+        QString("Warning, clock resolution is low (estimated as %1 ns)! Introspection "
```

Comment:
After seeing the screenshot: You could use `(estimated as %L1 ns)` to get localized number formatting. This should give you the appropriate thousands-separator (if appropriate at all)